data.sql

```diff
@@ -5,4 +5,41 @@ VALUES
 	(1, 'Agumon', 'Feb 3, 2020', 0, TRUE, 10.23),
 	(2, 'Gabumon', 'Nov 15, 2018', 2, TRUE, 8),
 	(3, 'Pikachu', 'Jan 7, 2021', 1, FALSE, 15.04),
-	(4, 'Devimon', 'May 12, 2017', 5, TRUE, 11)
+	(4, 'Devimon', 'May 12, 2017', 5, TRUE, 11),
+	(5, 'Charmander', 'Feb 8, 2020', 0, FALSE, -11),
+	(6, 'Plantmon', 'Nov 15, 2022', 2, TRUE, -5.7),
+	(7, 'Squirtle', 'Apr 2, 1993', 3, FALSE, -12.13),
+	(8, 'Angemon', 'Jun 12, 2005', 1, TRUE, -45),
+	(9, 'Boarmon', 'Jun 7, 2005', 7, TRUE, 20.4),
+	(10, 'Blossom', 'Oct 13, 1998', 3, TRUE, 17);
+
+/* Insert the following data into the owners table */
+INSERT INTO owners (full_name, age)
+VALUES 
+		('Sam Smith', 34),
+		('Jennifer Orwell', 19),
+		('Bob', 45),
+		('Melody Pond', 77),
+		('Dean Winchester', 14),
+		('Jodie Whittaker', 38);
+
+/* Insert the following data into the species table */
+INSERT INTO species (name)
+VALUES 
+		('Pokemon'),
+		('Digimon');
+
+/* Modify your inserted animals so it includes the species_id value: */
+UPDATE animals SET species_id = CASE WHEN name LIKE '%mon' THEN 2 ELSE 1 END;
```

Comment:
* Kindly use the right query here as the question says (`Modify your inserted animals so it includes the species_id value:
If the name ends in "mon" it will be Digimon
All other animals are Pokemon`) so your query can be written this way (`When name ends with mon ? species_id = Digimon else Pokemon).

Reply:
@Davidosky007 Thanks so much for the review.

I want to set a comment about this suggestion, you're saying that I should change the way I'm saving in the species_id to the strings Digimon or Pokemon but in the previous day we already have that with the species column and in this part of the exercise is required to make a foreign key with the species table so when I make the join the table displays the correct species for each animal. 

But if I do it in the way you're suggesting there's no need to even make the join to know with species name have each animal.

And also the exercise is requesting to make a relation between the id of the species so it has no sense to write the name as id if the relation is with a number.